tox.ini

```diff
@@ -1,12 +1,20 @@
 [tox]
-envlist=check,py{26,27,34,35,36,py}-pytest{28,29,30}
+envlist=check,py{26,27,34,35,36,py}
 
 [testenv]
 commands=py.test {posargs:testing/}
 deps=
-  pytest28: pytest~=2.8.0
-  pytest29: pytest~=2.9.0
-  pytest30: pytest~=3.0.0
+  pytest
```

Comment:
why completely remove those?

Reply:
I thought we agreed that it didn't make sense to run the tests using old pytest versions given that those versions vendor pluggy anyway.

Reply:
we should leave one with pytest 3.2 to have one "working" pytest version

Reply:
That's the pytest version we use for the normal test environments for all python versions, so we're covered. For example: https://travis-ci.org/pytest-dev/pluggy/jobs/270596649

Reply:
All the "normal" builds (Python 2.6 and TOEXENV=py26) executes with the latest pytest.